salesagility/SuiteCRM
# fix issue #517 - Check box field do not Export value in module Reports

Fixed #517 - Check box field do Export value in module Reports to csv.
## Description




## Motivation and Context


## How To Test This

1. Create report based on module which contain checkbox for example: Leads
2. Use to report field with checkbox ("Do not call" in this case"
3. Save 
4. Select: "Export" from "Actions" drop
5. Open exported .csv file
6. You can see "0" or "1" values in field column
## Types of changes


### Final checklist
